catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -366,4 +378,19 @@
         </plugins>
 
     </build>
+
+    <profiles>
+        <profile>
+            <id>webpack dev</id>
+            <activation>
+                <property>
+                    <name>skipStatic</name>
```

Comment:
Might be worth having the separate flags, but also a catch all flag for dev mode.